SonarQube.Common/AnalysisConfig/AnalysisConfig.cs

```diff
@@ -33,6 +33,12 @@ public class AnalysisConfig
         ///  See https://jira.sonarsource.com/browse/SONARMSBRU-100 for details.</remarks>
         public string SonarRunnerWorkingDirectory { get; set; }
```

Comment:
I'm closing the comment and leaving things as they are. There isn't enough evidence that a change is required.